presto-docs/src/main/sphinx/release/release-0.247.rst

```diff
@@ -0,0 +1,42 @@
+=============
+Release 0.247
+=============
+
+**Highlights**
+==============
+* Add the interface ``QueryInterceptor`` to allow for custom logic to be executed before or after query execution. (:pr:`15565`).
+* Add support for temporary (session-scoped) functions.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug for reporting output data sizes for optimized repartitioning.
+* Fix accounting for revocable memory that could cause some queries not to spill when they should.
+* Fix a race condition in enum key lookup which caused queries using :func:`enum_key` to crash occasionally with an internal error. (:pr:`15607`).
+* Add an implementation of :func:`array_intersect` that takes an array of arrays as input.
+* Add support for temporary (session-scoped) functions.
+* Add support for specifying session properties via regex matching on client info using :doc:`/admin/session-property-managers`.
+
+SPI Changes
+___________
+* Add a new field `Optional<ConnectorSplit> split` to ``ConnectorTableLayoutHandle#getIdentifier``.
+
+Hive Changes
+____________
+* Fix a bug that could cause queries to fail with ``HIVE_EXCEEDED_SPLIT_BUFFERING_LIMIT`` error when scanning large bucketed tables using grouped execution.
+* Add support for partition stats based optimization, including partition pruning and column domain stripping for fragment result caching.
+
+JDBC Changes
+____________
+* Add the interface ``QueryInterceptor`` to allow for custom logic to be executed before or after query execution. (:pr:`15565`).
+
+Verifier changes
+----------------
```

Comment:
the other sections above have underscores (`_`) and this line has hyphens (`-`) - lets move everything to underscores?